spec/policies/placements/user_policy_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "rails_helper"
+
+RSpec.describe Placements::UserPolicy do
+  subject { described_class.new(current_user, user).destroy? }
+
+  context "where current user and user are the the same user" do
+    let(:current_user) { create(:placements_user) }
+    let(:user) { current_user }
+
+    it "returns false, user cannot destroy themselves" do
+      expect(subject).to eq false
+    end
+  end
+
+  context "where current user and user are different users" do
+    let(:current_user) { create(:placements_user) }
+    let(:user) { create(:placements_user) }
+
+    it "retursn true, user can destroy other users" do
```

Comment:
```suggestion
    it "returns true, user can destroy other users" do
```

Reply:
So embarrassing when I'm usually the person complaining about other people's typos.